doc/releases/changelog-0.25.0.md

```diff
@@ -573,6 +538,14 @@ of operators.
 
 <h3>Improvements 📈</h3>
 
+* `default.qubit` now natively executes any operation that defines a matrix except
+  for trainable `Pow` operations. 
+  [(#2836)](https://github.com/PennyLaneAI/pennylane/pull/2836)
+  
+* When adjoint differentiation is requested, circuits are now decomposed so
+  that all trainable operations have a generator.
+  [(#2836)](https://github.com/PennyLaneAI/pennylane/pull/2836)
```

Comment:
Same PR link? 🤔

Reply:
I think the first item should have stayed in new features, before 2836 it was not possible to execute `Sum`, `Prod`. But it was already possible for `Adjoint`

Reply:
From a user perspective, I think it *reads* more like an improvement under the hood rather than a new feature. Is there  a way to reword that first item?

Reply:
It was possible to use symbolic operations but not to use them on `default.qubit`, so for me it is a new feature compared to 0.24.

Reply:
Okay let's see what @josh146 says when he gets a chance to review